packages/vue-renderer/src/renderers/ssr.js

```diff
@@ -82,14 +82,16 @@ export default class SSRRenderer extends BaseRenderer {
       m.style.text() +
       m.script.text() +
       m.noscript.text()
+    const needInjectScripts = this.options.render.injectScripts !== false
+    const needInjectState = this.options.render.injectState !== false
```

Comment:
In my requirements, I only need the data fetched at the server side and no scripts related to the vue. And the two options correspond to the two functions in custom template —— renderState and renderScripts

Reply:
> I think no extra option would be needed. Both are scripts and not usable standalone

But in my requirements, I only need the data fetched at the server side and no scripts related to the vue. And the two options correspond to the two functions in custom template —— renderState and renderScripts

Reply:
Agree with @pi0 , we'd better keep same API as vue-ssr-renderer which only covers `inject`.

For `needInjectState`, actually you can do some process in your store dir or asyncData, exclude data fetching in lower version browsers, this would be more efficient since it won't trigger data fetching request at all.